widgetssdk/src/main/java/com/glia/widgets/call/CallState.java

```diff
@@ -395,6 +402,8 @@ public static class Builder {
         private Engagement.MediaType requestedMediaType;
         private boolean isSpeakerOn;
         private boolean isOnHold;
+        //My be helpful when converting to Kotlin, as ide makes field nullable.
```

Comment:
I think there are some typos here:
My => May
ide => not sure what does it mean.

Reply:
fixed https://github.com/salemove/android-sdk-widgets/pull/402/commits/23f80355fe595bd0b69fa31ece8b37f85c3486f3

Reply:
@DavDo Ah, ide is Android Studio. 🙂 
If you wrote with capital letters (IDE) I would guess it. 🙂